OSDOCS#5564: Adds bugs and notes to 4.13 MicroShift RNs (#58946)
openshift/openshift-docs: microshift_release_notes/microshift-4-13-release-notes.adoc

//==== New feature for use during installation here
//can include a note about a change in base RHEL image

[id="microshift-4-13-support"]
=== Support

[id="microshift-4-13-sos-reports"]
==== {product-title} sos reports

> With this release you can run an `sos` report to collect troubleshooting information about a host. The reort will generate a detailed report with all the enabled plugins and data from the different components and applications in a system.

Reviewer: ```suggestion
With this release you can run an `sos` report to collect troubleshooting information about a host. The report will generate a detailed report with data from all the enabled plugins and data from the different components and applications in a system.
```

For more information, see xref:../microshift_support/microshift-sos-report.adoc#about-microshift-sos-reports_microshift-sos-report[About MicroShift sos reports].

[id="microshift-4-13-etcd"]
==== {product-title} etcd

> With this release, {product-title} etcd is delivered as part of the {product-title} RPM. The etcd service is run as a separate process and the lifecycle is managed automatically by {product-title}. You can gather `journalctl` logs to observe and debug the etcd server logs.

Reviewer: ```suggestion
With this release, {product-title} etcd is run as a separate process whose lifecycle is managed automatically by {product-title}. You can gather `journalctl` logs to observe and debug the etcd server logs.
```

etcd was always there, we're just running it differently.

> For more information, see xref:../microshift_support/microshift-etcd.html#microshift-observe-debug-etcd-server_microshift-etcd[MicroShift etcd].

Reviewer 2: This link seems to be going to the first module in the assembly: Observe and debug the MicroShift etcd server. Is that the intent? If so, maybe update the link text.

//[id="microshift-4-13-deprecated-removed-features"]
//== Deprecated and removed features

[id="microshift-4-13-bug-fixes"]
== Bug fixes

* Previously, {product-title} generated `kubeconfig` files using certificate authorities (CA) shared by all embedded components. With this update, the CAs and certificates are reconfigured to ensure the `kubeconfig` files are independent. (link:https://issues.redhat.com/browse/OCPBUGS-10223[*OCPBUGS#10223*])

> * Previously, the `systemd-resolved` configuration was used by kubelet as an alternative dns resolver, but the DNS Corefile configuration did not use the same `systemd-resolved` configuration. With this update, {product-title} uses network name resolution provided by `systemd-resolved` for both kublet and cluster DNS when `systemd-resolved.service` is enabled. (link:https://issues.redhat.com/browse/OCPBUGS-6786[*OCPBUGS-6786*])

Reviewer 2: s/dns resolver/DNS resolver
s/uses network name/uses the network name  (Not 100% sure on this one)
s/when `systemd-resolved.service`/when the `systemd-resolved.service`

//[id="microshift-4-13-deprecated-removed-features"]
//== Deprecated and removed features

[id="microshift-4-13-bug-fixes"]
== Bug fixes

> * Previously, the certificate authorities generated `kubeconfig` files for {product-title} embedded components. With this update, the embedded components are reconfigured to ensure the `kubeconfig` files are independent. (link:https://issues.redhat.com/browse/OCPBUGS-10223[*OCPBUGS#10223*])

Reviewer: ```suggestion
* Previously, {product-title} generated `kubeconfig` files using certificate authorities (CA) shared by all embedded components. With this update, the CAs and certificates are reconfigured to ensure the `kubeconfig` files are independent. (link:https://issues.redhat.com/browse/OCPBUGS-10223[*OCPBUGS#10223*])
```

* Previously, the `systemd-resolved` configuration was used by kubelet as an alternative dns resolver, but the DNS Corefile configuration did not use the same `systemd-resolved` configuration. With this update, {product-title} uses network name resolution provided by `systemd-resolved` for both kublet and cluster DNS when `systemd-resolved.service` is enabled. (link:https://issues.redhat.com/browse/OCPBUGS-6786[*OCPBUGS-6786*])

> * Previously, the `microshift-etcd` version was not specified in the output to multiple commands. With this update, `microshift-etcd` outputs the correct version. (link:https://issues.redhat.com/browse/OCPBUGS-8704[*OCPBUGS-8704*])

Reviewer: I'm not sure this one is worth mentioning. The bug was both introduced and fixed in 4.13.
